scripts/script.js

```diff
@@ -1,118 +1,92 @@
 $(function () {
 
-    var carouselList = $('#js-myCarousel ul'),
-        interval = setInterval(changeSlideLeft, 1000),
-        i = 0,
-        $activeSlide = $('.indicators .item');
+    var carouselList = $('#js-myCarousel ul');
+    var currentSlideIndex = 0;
+    var $indicators = $('.indicators .item');
+    var lastSlideIndex = $indicators.length - 1;
+    // var interval = setInterval(function() {
+    //     moveSlidesForward(1);
+    // }, 1000);
 
 //start karuzeli//
 
-    function changeSlideLeft() {
-        if (i === 4) {
-            clearInterval(interval);
-            interval = setInterval(changeSlideRight, 1000);
+    function moveSlidesForward(numberOfSlides) {
+        if (currentSlideIndex === lastSlideIndex) {
```

Comment:
to jest to samo co miales wczesniej jako `i === 4`; 
zamiast `i` dalem `currentSlideIndex` bo to lepsza nazwa wg mnie. Natomiast zamiast na sztywno wrzucac liczbe 4 to po prostu patrze ile jest indikatorow i wrzucam ostatni index w zmienna `lastSlideIndex`. Tym sposobem jesli dodasz slajdy i indikatory do htmla to nie musisz nic zmieniac w skrypcie.